js/apps/account-ui/maven-resources/theme/keycloak.v3/account/index.ftl

```diff
@@ -104,11 +104,7 @@
     <div id="app">
       <main class="container">
         <div class="keycloak__loading-container">
```

Comment:
Let's also remove any custom classes and styling that are written here. Including the styles in the tag above and start clean. Then afterwards we can see what [critical CSS](https://web.dev/articles/extract-critical-css) can be extracted from PatternFly to optimize loading after.